src/SmartComponents/Actions/ListActions.js

```diff
@@ -154,11 +157,40 @@ class ListActions extends Component {
         });
     }
 
+    onNavigate(_event, _item, key) {
+        const { history } = this.props;
+        history.go(-key);
+    }
+
+    parseBreadcrumbs(breadcrumbs, params) {
```

Comment:
If there is a possibility this might be used again, parsing the breadcrumb title, we ought to abstract the logic into a helper or a functional component...